README.md

```diff
@@ -96,9 +96,9 @@ Want to log with Aardvark but don’t want to use Aardvark’s bug reporting too
 
 ## Requirements
 
-* Xcode 11.0 or later
```

Comment:
`ci.yml` is still using Xcode_11.7. We probably wanna update that 🙂

Reply:
🤔 Pretty sure it was using Xcode 13. In any case, should be Xcode 14 now. 😄

Reply:
🤦‍♂️ apparently I was quite behind. My bad!